ckb-bench/Cargo.toml

```diff
@@ -20,3 +20,6 @@ toml = "0.5"
 lru = "0.6.0"
 crossbeam-channel = "0.5.1"
 git-version = "0.3.5"
+tokio = { version = "1.14.0", features = ["full"] }
+futures = "0.3"
+rayon = "1.5"
```

Comment:
I didn't see rayon was required in codebase?

Reply:
i will remove it